nested-passthrough/ansible/setup_rhelai/tasks/cleanup_rhelai.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Delete RHELAI vm
```

Comment:
Just so I understand, this is not going to be invoked by default right ? As it's not part of the setup_rhelai.yml playbook [0]

[0] https://github.com/openstack-k8s-operators/rhoso-rhelai/blob/main/nested-passthrough/ansible/setup_rhelai.yml

Reply:
Ah forget this, I just now saw https://github.com/openstack-k8s-operators/rhelai-validation/pull/2/files

So this is a dependency on that. Got it

Reply:
Yeah, just a way to expose the cleanup task from ansible instead of the Makefile